app/src/androidTest/java/ch/epfl/sweng/swenggolf/ProfileActivityOtherUserTest.java

```diff
@@ -47,4 +54,48 @@ public void editButtonGone() {
                 ViewMatchers.Visibility.GONE)));
     }
 
+    @Test
+    public void canFollow() {
+        testToast("You are now following ");
+
+        ValueListener<String> listener = new ValueListener<String>() {
+            @Override
+            public void onDataChange(String value) {
+                assertNotNull(value);
+            }
+
+            @Override
+            public void onCancelled(DbError error) {
+                fail();
+            }
+        };
+
+        Database.getInstance().read("/followers/" + user.getUserId(), otherUser.getUserId(),
```

Comment:
You put constants for the path in `Database`, shouldn't you use `FOLLOWER_PATH` instead?

Reply:
It's done.